src/app-simplefin/app-simplefin.js

```diff
@@ -333,6 +337,12 @@ async function getAccounts(accessKey, startDate, endDate) {
 
   params.push(`pending=1`);
 
+  if (accounts) {
+    accounts.forEach((id) => {
+      params.push(`account=${id}`);
+    });
+  }
+
```

Comment:
Although the existing account IDs do not require URI encoding, to future-proof it (in case the structure of an account ID changes down the road) I am accepting this suggestion.